### ISSUE TITLE
Перерегистрация. Смена имени компании

### ISSUE THREAD

**Issue by mifkamaz** [shtrih-m/javapos_shtrih#43]
Как при перерегистрации изменить:
1. Имя/адрес компании
2. Флаг азартные игры
3. Флаг лотерея
4. Адрес сайта ФНС

**Comment by nyxiscoo1** [comment]
Здравствуйте. По возможности я бы рекомендовал использовать мастер фискализации.
Если же хотите сами, то 1 и 4 меняются через таблицу в ФР.
2 и 3 зависит от модели устройства, версии прошивки. [Вот класс](https://github.com/shtrih-m/javapos_shtrih/blob/master/Source/android/tinyJavaPosTester/src/com/shtrih/tinyjavapostester/Fiscalizer.java) демонстрирующий как выполнить перерегистрацию с установкой всех параметров, список поддерживаемого оборудования указан в шапке, если нужно больше оборудования, например, мобайл 20019, то нужно добавить еще искусственного интеллекта.

**Comment by nyxiscoo1** [comment]
Да, есть какая-то такая утилита, можно скачать [вот тут](https://exam.shtrih-m-partners.ru/base/), но я не уверен, что она поддерживает все устройства, я ей никогда не пользовался.

**Comment by mifkamaz** [comment]
Использую Fiscalizer
При записи ИНН ОФД происходит ошибка `51, Некорректные параметры в команде`
Вызов `writeTable(14, 1, 12, "7709364346  ")`

**Comment by mifkamaz** [comment]
`fsReFiscalization(inn, rnm, taxSystemCode, operationMode, params.ReasonCode)`
Cрабатывает если только `taxSystemCode = 1 && operationMode == 1`
В остальных случаях: `51, Некорректные параметры в команде`

**Comment by nyxiscoo1** [comment]
Какая модель  устройства? Какая версия прошивки?

**Comment by nyxiscoo1** [comment]
А версия прошивки из запроса длинного статуса?

**Comment by mifkamaz** [comment]
Это все доступные поля в этом объекте

**Comment by nyxiscoo1** [comment]
В этом объекте да, есть другой метод  printer.readLongPrinterStatus().getFirmwareBuild().
У меня правда под рукой сейчас нет мобайла, но я на нем наблюдал такое поведение, что он оказывался регистрироваться с такой же ошибкой, если ему передавать флаги "Применение в Интернет", "Режим БСО", возможно он так же реагирует на "Применение в сфере услуг", которое Вы указываете.
